analyses/immune-deconv/03-quantiseq-at-threshold.Rmd

```diff
@@ -0,0 +1,253 @@
+---
+title: "Assess quanTIseq results calculated with tumor purity thresholded data"
+author: "SJ Spielman for CCDL"
+date: "2023"
+output: 
+  html_notebook:
+  toc: TRUE
+toc_float: TRUE
+---
+
+```{r setup, include = FALSE}
+library(tidyverse)
+theme_set(theme_bw())
+set.seed(2023)
+```
+
+
+This notebook was written as part of manuscript revisions.
+Here, we explore how `quanTIseq`-related results might change if only tumors at a certain level of known tumor purity (`tumor_fraction` metadata variable) are considered.
+Only stranded RNA data is considered here, as in the manuscript.
+
+The associated issue for this exploration is here: https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1628
+
+
+
+## Set directories and files
+
+```{r}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+palette_file <- file.path(root_dir, "figures", "palettes", "broad_histology_cancer_group_palette.tsv")
+metadata_file <- file.path(data_dir, "pbta-histologies.tsv")
+quantiseq_file <- file.path("results", "quantiseq_deconv-output-thresholded.rds")
+
+
+```
+
+
+Read in and prep data:
+
+```{r}
+quantiseq_raw_df <- read_rds(quantiseq_file)
+metadata_df <- read_tsv(metadata_file, guess_max = 10000)
+palette_df <- read_tsv(palette_file)
+
+# These are the groups we highlight in the manuscript
+cancer_groups_of_interest <- c("Pilocytic astrocytoma", 
+                               "Diffuse midline glioma",
+                               "Craniopharyngioma", 
+                               "Ganglioglioma",
+                               "Ependymoma", 
+                               "Medulloblastoma",
+                               "Schwannoma",
+                               "Neurofibroma Plexiform",
+                               "Other embryonal tumor",
+                               "Atypical Teratoid Rhabdoid Tumor",
+                               "Meningioma",
+                               "Dysembryoplastic neuroepithelial tumor")
+
+# Combine with cancer group and palette info
+quantiseq_df <- quantiseq_raw_df %>%
+  select(-library, -method) %>%
+  rename(Kids_First_Biospecimen_ID = sample) %>%
+  inner_join(
+    select(
+      metadata_df, 
+      cancer_group, 
+      Kids_First_Biospecimen_ID
+    )
+  ) %>%
+  inner_join(
+    select(
+      palette_df, 
+      cancer_group,
+      cancer_group_display, 
+      cancer_group_hex
+    )
+  ) %>%
+  select(-cancer_group) %>%
+  # Filter to relevant rows as in MS figure
+  filter(cell_type != "uncharacterized cell") %>%
+  filter(cancer_group_display %in% cancer_groups_of_interest)
+```
+
+## Re-generate plots
```

Comment:
Would it be possible to pull in the old plots here? I think the fastest way to go about this might be something like:



```
![](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/586d878b25c80495a2541292c2d9b289167f6ec9/figures/pngs/figure5.png)
```

And repeat it with Figure S6